### PR TITLE
Fix plural/singular

### PR DESCRIPTION
**Pull request by xnorly**
When upgradableCount is 1, the banner should use singular: "You have 1 extension that needs"
Otherwise the banner should use plural: "You have 2 extensions that need"

### CODE REVIEW

File: packages/page-accounts/src/Accounts/BannerExtension.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ function BannerExtension (): React.ReactElement | null {
 
       return (
         <Banner type='warning'>
-          <p>{t<string>('You have {{upgradableCount}} extensions that need to be updated with the latest chain properties in order to display the correct information for the chain you are connected to. This update includes chain metadata and chain properties.', { replace: { upgradableCount } })}</p>
+          <p>{t<string>(`You have ${upgradableCount} ${upgradableCount===1? "extension that needs" : "extensions that need"} to be updated with the latest chain properties in order to display the correct information for the chain you are connected to. This update includes chain metadata and chain properties.`)}</p>
```
[the remark this answers is not in the document]
Comment (jacogr) [reply packages/page-accounts/src/Accounts/BannerExtension.tsx:43]: Something like -

```js
upgradableCount===1
  ? t('You have 1 extension that nesds to be upgraded')
  : t('You have {{upgradableCount}} extensions that needs to be upgraded', { replace: { upgradableCount }})
```

[the remark this answers is not in the document]
Comment (xnorly) [reply packages/page-accounts/src/Accounts/BannerExtension.tsx]: Hey @jacogr 

Thank you for your help!
I implemented it in the last commit 01f0e54
Hope it's ok.

Happy new year

[the remark this answers is not in the document]
Comment (blckshdwstudio) [reply packages/page-accounts/src/Accounts/BannerExtension.tsx]: Hello! The letter "л
л
" is superfluous in translation into Russian

File: packages/page-accounts/src/Accounts/BannerExtension.tsx [hunk 2]
```diff
@@ -40,7 +40,13 @@ function BannerExtension (): React.ReactElement | null {
 
       return (
         <Banner type='warning'>
-          <p>{t<string>('You have {{upgradableCount}} extensions that need to be updated with the latest chain properties in order to display the correct information for the chain you are connected to. This update includes chain metadata and chain properties.', { replace: { upgradableCount } })}</p>
+          <p>
+            {upgradableCount===1
```
Comment (jacogr) [inline packages/page-accounts/src/Accounts/BannerExtension.tsx:44]:
```suggestion
            {upgradableCount === 1
```

Fix linting.

**Review by jacogr — APPROVED** [review]
Thank you!

**Comment by xnorly** [comment]
> Thank you!

Thank you for your help!

**Pull request merged**